khelo/functions/src/index.js

```
@@ -66,6 +66,18 @@ exports.teamPlayerChangeObserver = (0, firestore_2.onDocumentUpdated)({region: R
   await teamService.notifyOnAddedToTeam(oldTeam, newTeam);
 });
 
+exports.userDeleteObserver = (0, firestore_2.onDocumentDeleted)({region: REGION, document: "users/{userId}"}, async (event) => {
+  const snapshot = event.data;
+  if (!snapshot) {
+    (0, logger.error)("No data associated with the event");
+    return;
+  }
+  const user = snapshot.data();
+  if (user && user.id) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)